pokete.py

```diff
@@ -1389,9 +1382,30 @@ def map_additions():
 # Actual code execution
 #######################
 if __name__ == "__main__":
-    do_logging, load_mods, force_pynput = parse_args(sys.argv)
+    do_logging, load_mods = parse_args(sys.argv)
     # deciding on wich input to use
-    if sys.platform == "linux" and not force_pynput:
+    if sys.platform == "win32":
+        import msvcrt
+
+
+        def recogniser():
+            """Gets keyboard input from pynput"""
```

Comment:
```suggestion
            """Gets keyboard input from msvcrt, the Microsoft Visual C++ Runtime"""
```

Reply:
Oh yes, I forgot about the comments.
Then please approve.